Library/Homebrew/keg.rb

```diff
@@ -339,14 +339,14 @@ def to_formula
   end
 
   def installed_dependents
+    return [] unless linked?
```

Comment:
Yep. Anything without an opt-link can be considered outdated or just plain broken.